### PR TITLE
Lint scss files

### PR DESCRIPTION
**Pull request by recmo**

### CODE REVIEW

File: src/styles/colors.scss [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 
 // TODO: decode how do we import variables by scss import or css-modules composes
 .footer {
   color: $color5;
 }
+
+// Gradients
+// sass-lint:disable no-color-literals
+$gradient1: linear-gradient(180deg, #09719b 0%, #09719b 2.36%, #424a52 100%);
+// sass-lint:enab
```
Comment (recmo) [inline src/styles/colors.scss]: Line got cut half way

File: src/components/Footer.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.Footer {
-  height: 1rem;
-  composes: Montserrat from '../styles/fonts.scss';
```
Comment (banciur) [inline src/components/Footer.scss]: As we talked on slack we shouldn't use `composes` feature of css-modules. Same goes for Header.scss file

**Review by banciur — CHANGES_REQUESTED** [review]
Now when I'm trying to build I receive 

```
ERROR in ./~/css-loader?{"camelCase":"dashesOnly"}!./~/sass-loader/lib/loader.js!./src/components/Footer.scss
Module build failed: 
  @extend %footer;
         ^
      ".footer" failed to @extend "%footer".
The selector "%footer" was not found.
Use "@extend %footer !optional" if the extend should be able to fail.
      in /home/banciur/projects/neufund/generic-ico/src/components/Footer.scss (line 5, column 11)
 @ ./src/components/Footer.scss 4:14-134 18:2-22:4 19:20-140
```
I thought there is some problem with dependencies so I deleted `node_modules` directory and performed `yarn` command and then `yarn serve` but it didn't help

File: src/components/Footer.scss (hunk 2 above)
Comment (recmo) [reply src/components/Footer.scss]: This is an old version of the file. Are you sure you reviewed the latest version?

File: src/components/Footer.scss [hunk 3]
```diff
@@ -1,9 +1,11 @@
-.Footer {
+@import '../styles/fonts';
```
Comment (banciur) [inline src/components/Footer.scss]: You are missing `@import '../styles/colors';` in Footer.scss

File: src/styles/colors.scss [hunk 4]
```diff
@@ -1,18 +1,23 @@
...
+$color8: #9caccc;
 $color9: #032533;
-$color10: #686E75;
+$color10: #686e75;
 $color11: #315568;
-$color12: #B3B7Ba;
+$color12: #b3b7ba;
 
 // TODO: decode how do we import variables by scss import or css-modules composes
 .footer {
```
Comment (banciur) [inline src/styles/colors.scss]: This line should look like this `%footer {`

**Review by banciur — APPROVED** [review]
It is working at this moment.

File: src/components/Button.scss [hunk 5]
```diff
@@ -1,15 +1,20 @@
...
+  text-align: center;
   line-height: 40px;
-  border:none;
-}
+  color: $text;
 
-.Button:hover {
-  background-color: #09769F;
-  outline: none;
+  :hover {
```
Comment (recmo) [inline src/components/Button.scss:16]: Turns out this should have been `&:hover`.

**Review by recmo — COMMENTED** [review]
Bug.

**Pull request merged**